src/renderer/components/VideoAligner/VideoAligner.tsx

```diff
@@ -254,6 +276,11 @@ export default function VideoAligner({ video }: Props) {
             <FormControl>
               <FormLabel>Name</FormLabel>
               <Input
+                onKeyDown={(e) => {
+                  if (e.key === 'Enter') {
+                    handleClose();
+                  }
+                }}
```

Comment:
I don't think you'll need this *and* the hotkeys code (they're both doing the same thing).

Pretty sure you could remove this and leave the `useHotkeys` - could be wrong though